packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -153,34 +140,67 @@ function FontCollection( { id } ) {
 		setFontsToInstall( [] );
 	};
 
+	const handleFontDownload = async ( fontFamily ) => {
+		return Promise.all(
```

Comment:
Based on the last discussion we had about async, does this need to await?

```suggestion
		return await Promise.all(
```